cmd/archeio/app/buckets.go

```diff
@@ -29,8 +30,11 @@ import (
 func awsRegionToS3URL(region string) string {
 	// for now always return @ameukam's test bucket
```

Comment:
Maybe remove this comment?

Reply:
Thanks for noticing it, I've removed it in b78adbe